README.rst

```diff
@@ -152,12 +152,12 @@ If the exit code of the executable script is higher than ``0`` the test will be
 Unmanaged Configurations
 ------------------------
 
-In some cases it is necessary to ensure that some configuration sections won't be
+In some cases it could necessary to ensure that some configuration sections won't be
```

Comment:
I think it's `could be`

Reply:
I corrected the sentence and added a sentence about the unmanaged settings not being in the default configuration.